packages/ui/src/TapToEdit.tsx

```diff
@@ -236,7 +238,7 @@ export const TapToEdit = ({
                 </Text>
               )}
             </Box>
-            {editable && (
+            {editable && fieldProps?.type !== "textarea" && (
```

Comment:
I'll chat with Jo! Going to merge as-is, as that's how it was set up previously. It maybe makes sense in the middle, because the text can grow arbitrarily large.